tests/phpunit/tests/functions/sendFrameOptionsHeader.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Tests for the send_frame_options_header function.
+ *
+ * @group functions.php
+ *
+ * @covers ::send_frame_options_header
+ */
+class Tests_functions_sendFrameOptionsHeader extends WP_UnitTestCase{
+
+
+	/**
+	 * Just test for function
+	 *
+	 * @ticket 59851
+	 */
+	public function test_send_frame_options_header() {
+
+		$this->assertTrue( function_exists('send_frame_options_header') );
```

Comment:
This is not really a useful test in itself.

To make this test useful, use `xdebug_get_headers()` and `@runInSeparateProcess` to check that the correct header was sent. See existing core tests for precedence